app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
```

Comment:
Not 100 % sure, but I think according to conventions you should separate imports with blank lines if they are from different blocks (Standard Library Imports, Third-Party Library Imports, Local Imports). Both errors and cafe are local imports, so maybe no need to blank line

Reply:
yeap, agree